tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/bir_test_project/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org= "bir"
+name= "objs"
+version= "0.1.0"
+exported=["serv_classes"]
```

Comment:
Is this required?

Reply:
We need to have a `Ballerina.toml`. But we don't need a value for the `exported` key.

Reply:
Yes, I meant this line. :)